app/code/Magento/GraphqlPlayground/Controller/Index/Index.php

```diff
@@ -0,0 +1,84 @@
+<?php declare(strict_types=1);
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\GraphqlPlayground\Controller\Index;
```

Comment:
For consistency module name should be changed.
```suggestion
namespace Magento\GraphQlPlayground\Controller\Index;
```